Doc/library/atexit.rst

```diff
@@ -14,12 +14,11 @@ functions.  Functions thus registered are automatically executed upon normal
 interpreter termination.  :mod:`atexit` runs these functions in the *reverse*
 order in which they were registered; if you register ``A``, ``B``, and ``C``,
 at interpreter termination time they will be run in the order ``C``, ``B``,
-``A``.
-
-**Note:** The functions registered via this module are not called when the
+``A``.  The functions registered via this module are not called when the
```

Comment:
Maybe leave the Note, it's unrelated to the current bpo issue.

Reply:
Ok, I reverted that change.